docs/src/library-js.md

```diff
@@ -5,30 +5,27 @@ title: "Library"
 
 Playwright Library provides unified APIs for launching and interacting with browsers, while Playwright Test provides all this plus a fully managed end-to-end Test Runner and experience.
 
-Under most circumstances, for end-to-end testing, you'll want to use `@playwright/test` (Playwright Test), and not `playwright` (Playwright Library) directly. To get started with Playwright Test, follow its [Getting Started Guide](./intro.md).
-
-<!-- TOC -->
-- [Release notes](./release-notes.md)
+Under most circumstances, for end-to-end testing, you'll want to use `@playwright/test` (Playwright Test), and not `playwright` (Playwright Library) directly. To get started with Playwright Test, follow the [Getting Started Guide](./intro.md).
 
 ## When Should Playwright Library Be Used Directly?
 
-- creating an integration for a third party test runner (e.g. the third-party runner plugins listed [here](./test-runners.md) are built on top of Playwright Library)
-- automation and scraping
+- Creating an integration for a third party test runner. For example, third-party runner plugins listed [here](./test-runners.md) are built on top of the Playwright Library.
+- Automation and scraping.
 
-## Differences
+## Differences when using library
 
 ### Library Example
 
 The following is an example of using the Playwright Library directly to launch Chromium, go to a page, and check its title:
 
 
 ```js tab=js-ts
-import playwright, { devices } from 'playwright';
+import { chromium, devices } from 'playwright';
```

Comment:
whats the reason for this change? Just curious. Since tree-shaking isn't a thing anyway.